flask/launcher/src/main/java/net/corda/flask/launcher/JavaProcessBuilder.java

```diff
@@ -164,9 +165,12 @@ public ProcessBuilder build() {
             Path argumentFile = Files.createTempFile(PROCESS_BUILDER_PREFIX, ".arg");
             Runtime.getRuntime().addShutdownHook(new Thread(new Runnable() {
                 @Override
-                @SneakyThrows
                 public void run() {
-                    Files.delete(argumentFile);
+                    try {
+                        Files.delete(argumentFile);
+                    } catch (IOException ioe) {
+                        log.error(ioe.getMessage(), ioe);
```

Comment:
Thanks for doing this - I _was_ going to let this pass if it didn't cause a bug :grin::

However, I think logging frameworks must shut themselves down too - see Log4J's [LogManager.shutdown](https://logging.apache.org/log4j/2.x/log4j-api/apidocs/org/apache/logging/log4j/LogManager.html#shutdown--) which suggests that you cannot rely on the logging framework still to be running at this point. Perhaps log to `stderr` too, just in case?

Reply:
done